rpm/libseccomp.spec

```
@@ -42,7 +42,7 @@ for libseccomp.
 %build
 %autogen
 %configure --disable-static
```

Can be replaced with
```suggestion
%reconfigure --disable-static
```

---

Changed